android/timodule.xml

```diff
@@ -7,5 +7,8 @@
 	<iphone>
 	</iphone>
 	<android xmlns:android="http://schemas.android.com/apk/res/android">
+		<manifest>
+			<uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Don't we add this permission [already](https://github.com/appcelerator/titanium_mobile/blob/master/android/cli/commands/_build.js#L3615) ?